MemoryManagement/src/memorymanagementAlgorithm/Memory.java

```diff
@@ -1,47 +1,60 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
 package memorymanagementAlgorithm;
 
 import java.util.*;
+import javafx.scene.paint.Color;

```

Comment:
this import never used. please remove it.

Reply:
gonna fix it, fair enough.